tests/kernel/interrupt/prj.conf

```diff
@@ -1,4 +1,3 @@
 CONFIG_ZTEST=y
-CONFIG_IRQ_OFFLOAD=y
```

Comment:
This feature is not used in the test, correct?

Reply:
Correct. It used to be, but not anymore.